### PR TITLE
[pt-br] Add /docs/reference/setup-tools

### PR DESCRIPTION
**Pull request by MrErlison**
Add directory:
- content/pt-br/docs/reference/setup-tools/
- content/pt-br/docs/reference/setup-tools/kubeadm

Add files:
- content/pt-br/docs/reference/setup-tools/_index.md
- content/pt-br/docs/reference/setup-tools/kubeadm/_index.md

### CODE REVIEW

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: suggestion: Desenhado para se preocupar apenas com a inicialização e não com o provisionamento de máquinas.

Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: could add "úteis" after complementos.

Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: não está no escopo.
could be plural "estão"

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:31]: could add an "do" after versão.

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:29]: certificados do Kubernetes

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:28]: suggestion: para reverter quaisquer alterações feitas, neste host, pelo `kubeadm init` ou `kubeadm join`

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:27]: suggestion: para gerenciar os tokens pelo `kubeadm join`

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:26]: sugegstion:
para configurar seu cluster pelo `kubeadm upgrade`

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:23]: control plane = camada de gerenciamento
para inicializar um nó pela camada de gerenciamento do Kubernetes

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:15]: Em vez disso, esperamos que ferramentas de alto nível e mais personalizadas sejam construídas em cima do kubeadm e, idealmente, usando o kubeadm como base de todas as implantações, tornará mais fácil a criação de clusters em conformidade.

File: content/pt-br/docs/reference/setup-tools/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/_index.md:2]: Could be "Ferramentas de Configuração"

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/_index.md:2]: Could be: "Geradas pelo Kubeadm"

File: content/pt-br/docs/reference/setup-tools/_index.md
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/_index.md:2]: Could be:
"Ferramentas de Configuração"

**Review by DonatoHorn — COMMENTED** [review]
Good job Erlinton.

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: I changed "Desenhado" to "Projetado". That's ok?

Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: I think it's singular, because the direct sentence is "a instalação não está no escopo". Are you sure that?

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:5]: Keep "concept" here.

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:13]: The sentence starting with "Projetado para se preocupar..." reads disjointed to me. Maybe we can link it to the previous sentence?
Example: "... em funcionamento, e foi projetado para se preocupar apenas com a inicialização...".

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:15]: Missing "do"/"de" after "sejam construídas em cima".
Although the original sentence is "will make it easier", "tornará" is not really idiomatic Portuguese in this sentence. I think "torna" might read and sound better.

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/_index.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/_index.md:2]: "kubeadm (auto-gerado)" is more accurate here.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @MrErlison, thanks for your contribution!

Please find some change suggestions below.

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (stormqueen1990) [reply content/pt-br/docs/reference/setup-tools/kubeadm/_index.md]: Agree that "não está no escopo" should be singular since it refers to "instalação".

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:32]: We can update these links to point to `pt-br` for the pages you already have PRs for. I'll put a hold label on this one until we can get those other ones merged, so the index page has the correct links.

**Comment by stormqueen1990** [comment]
/hold

Until #33112, #33113, #33114, #33115, #33116, and #33117 are merged, so we can update the index links accordingly.

**Comment by MrErlison** [comment]
> /hold
> 
> Until #33112, #33113, #33114, #33115, #33116, and #33117 are merged, so we can update the index links accordingly.

These PR are broken links on `whatsnext`. When are we going to fix these files?

**Comment by stormqueen1990** [comment]
> These PR are broken links on `whatsnext`. When are we going to fix these files?

Those PRs created the missing files, correct? When they get merged, you will be able to rebase this branch and fix the links.

**Comment by rikatz** [comment]
/approve
Once the files are fixed, @stormqueen1990 feel free to lgtm 

I've just approved the other ones!

Thank you both for the hard work here!

**Comment by divya-mohan0209** [comment]
Hello @stormqueen1990 , could you please have a look at this PR and provide your LGTM whenever everything's okay per expectations?

**Comment by stormqueen1990** [comment]
Hi there, @divya-mohan0209!

Sure thing, will do! It will be blocked a little bit longer though as we are still missing the translations for `kubeadm init` and `join`.

File: content/pt-br/docs/reference/setup-tools/kubeadm/_index.md
Comment (jihoon-seo) [inline content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:25]: Among these links, the followings return 404 Page not found.

- https://kubernetes.io/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join
- https://kubernetes.io/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-upgrade

Comment (stormqueen1990) [reply content/pt-br/docs/reference/setup-tools/kubeadm/_index.md:25]: Hi there, @jihoon-seo!

Thanks for pointing it out! We're holding this PR until all the linked pages get translated.

**Comment by divya-mohan0209** [comment]
Hello @stormqueen1990 , good day! Please don't consider this me imposing or overstepping on your localization efforts. However, it seems that this PR has been pending merge due to localization efforts for other pages that have currently been merged. Please could you advise if this one can now be reviewed/approve appropriately?

**Comment by stormqueen1990** [comment]
> Hello @stormqueen1990 , good day! Please don't consider this me imposing or overstepping on your localization efforts. However, it seems that this PR has been pending merge due to localization efforts for other pages that have currently been merged. Please could you advise if this one can now be reviewed/approve appropriately?

Hi there, @divya-mohan0209! We're almost ready to merge this one. The last related PR is #38625, which is well on its way.

**Comment by stormqueen1990** [comment]
/unhold
/lgtm
/label tide/merge-method-squash

Thank you very much @MrErlison! 🎉

**Pull request merged**
